packages/contracts/test/helpers/LiveTestHelper.sol

```diff
@@ -25,10 +25,11 @@ import "../../src/dollar/ExcessDollarsDistributor.sol";
 import "../../src/dollar/SushiSwapPool.sol";
 import "../../src/dollar/interfaces/IERC1155Ubiquity.sol";
 
-import "Uniswap/v2-periphery/contracts/interfaces/IUniswapV2Router02.sol";
-import "Uniswap/v2-core/contracts/interfaces/IUniswapV2Factory.sol";
+import
+    "../../lib/Uniswap/v2-periphery/contracts/interfaces/IUniswapV2Router02.sol";
+import "../../lib/Uniswap/v2-core/contracts/interfaces/IUniswapV2Factory.sol";
 
-import "forge-std/Test.sol";
+import "../../lib/forge-std/src/Test.sol";
```

Comment:
This relative import doesn't seem right.

Reply:
That would be wrong. `import` is getting handled by remapping.txt. please revert it @3commascapital

Reply:
![Screenshot 2022-11-14 at 21 15 18](https://user-images.githubusercontent.com/90629478/201817964-269e804d-357c-4f52-9117-989c584a4d61.png)

Reply:
its just related to vscode settings but build works fine ser so not an issue

Reply:
yeah, i am not sure why, but linting fails without it. i am going to remove for now